src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -366,6 +375,36 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             };
         }
 
+        internal async Task<JObject> GetHostJsonExtensionsAsync()
+        {
+            var defaultJObject = new JObject();
+            var hostOptions = _applicationHostOptions.CurrentValue.ToHostOptions();
+            string hostJsonPath = Path.Combine(hostOptions.RootScriptPath, ScriptConstants.HostMetadataFileName);
+            try
+            {
+                _logger.LogInformation("Reading host.json for SyncTrigger.");
+                var hostJson = JObject.Parse(await FileUtility.ReadAsync(hostJsonPath));
+                if (hostJson.TryGetValue("extensions", out JToken token))
+                {
+                    return (JObject)token;
+                }
+                else
+                {
+                    return defaultJObject;
+                }
+            }
+            catch (JsonException ex)
+            {
+                _logger.LogWarning($"Unable to parse host configuration file '{hostJsonPath}'. : {ex}");
+                return defaultJObject;
+            }
+            catch (Exception ex) when (ex is FileNotFoundException || ex is DirectoryNotFoundException)
```

Comment:
Remove this file not found warning completely. Presence of the host.json file isn't mandatory so we don't want to create unnecessary warnings. See the other code which does a FileUtility.FileExists(hostJsonPath) check. 

Also, you should just return null if no host.json file or no extensions node found in the file, and have the calling code only add the extensions property when needed.

Reply:
I introduce the change as you requested. Thank you for the review!